### PR TITLE
[chore]: Document logs_index of OpensearchExporter

### PR DESCRIPTION
**Pull request by arpitjindal97**
**Documentation:** Mention `logs_index` field in README

### CODE REVIEW

**Comment by arpitjindal97** [comment]
@bryan-aguilar @atoulme Please review

**Comment by andrzej-stencel** [comment]
Can any of the code owners review? @Aneurysm9, @MitchellGale, @MaxKsyunz, @YANG-DB

**Review by MaxKsyunz — APPROVED** [review]

**Comment by arpitjindal97** [comment]
I don’t think @Aneurysm9 is active on this project

**Review by MitchellGale — APPROVED** [review]

File: exporter/opensearchexporter/README.md [hunk 1]
```diff
@@ -20,9 +20,15 @@ The documents are sent using [observability catalog](https://github.com/opensear
 
 ## Configuration options
 ### Indexing Options
+The Observability indices would follow the recommended for immutable data stream ingestion pattern using
```
Comment (andrzej-stencel) [inline exporter/opensearchexporter/README.md:23]: I'm not sure I understand this sentence. Should this be this?

```suggestion
The Observability indices would follow the recommended pattern for immutable data stream ingestion using
```

Comment (arpitjindal97) [reply exporter/opensearchexporter/README.md:23]: You are right

**Comment by arpitjindal97** [comment]
@andrzej-stencel Can we merge it?

**Review by YANG-DB — APPROVED** [review]

**Comment by arpitjindal97** [comment]
a gentle reminder

**Review by atoulme — APPROVED** [review]

**Comment by YANG-DB** [comment]
Yes please merge

**Review by TylerHelmuth — APPROVED** [review]

**Pull request merged**